dali/operators/image/resize/resampling_attr.cc

```diff
@@ -93,16 +94,6 @@ void ResamplingFilterAttr::PrepareFilterParams(
       }
       return false;
     };
-    if ((has_min && is_linear_default(make_cspan(min_arg_))) ||
-        (has_interp && is_linear_default(make_cspan(interp_type_arg_)))) {
-      static std::once_flag linear_default_warning_flag;
-      std::call_once(linear_default_warning_flag, [&]() {
-        DALI_WARN(
-            "The default behavior for LINEAR interpolation type has been changed to apply an "
-            "antialiasing filter. If you didn't mean to apply an antialiasing filter, please use "
-            "`antialias=False`");
-      });
-    }
```

Comment:
This was lamentably outdated.

Reply:
I have mixed filling about including this in this particular PR. But up to you.